types/react-modal/react-modal-tests.tsx

```diff
@@ -14,7 +14,7 @@ class ExampleOfUsingReactModal extends React.Component {
     const onAfterOpenFn = () => { };
     const onAfterCloseFn = () => { };
     const onRequestCloseFn = (event: React.MouseEvent | React.KeyboardEvent) => { };
-    const customStyle = {
+    const customStyle: { overlay: React.CSSProperties, content: React.CSSProperties } = {
```

Comment:
Can we change this, for readability:
```ts
const customStyle: ReactModal.Styles = {
```
I doubt I'd ever redeclare styles shape instead of importing declared type

Reply:
Thanks for catching this! Resolved in latest commit.